Alignment/OfflineValidation/macros/FitPVResiduals.C

```diff
@@ -4277,18 +4333,22 @@ params::measurement getTheRangeUser(TH1F *thePlot, Limits *lims, bool tag)
   } else {
     if (theTitle.Contains("means")) {
       if (theTitle.Contains("dxy") || theTitle.Contains("dx") || theTitle.Contains("dy")) {
-        if (theTitle.Contains("phi") || theTitle.Contains("pT") || theTitle.Contains("ladder")) {
+        if (theTitle.Contains("phi") || theTitle.Contains("ladder")) {
           result = std::make_pair(-lims->get_dxyPhiMax().first, lims->get_dxyPhiMax().first);
         } else if (theTitle.Contains("eta") || theTitle.Contains("mod")) {
           result = std::make_pair(-lims->get_dxyEtaMax().first, lims->get_dxyEtaMax().first);
+        } else if (theTitle.Contains("pt")) {
```

Comment:
shouldn't this be (here and elsewhere)?

```suggestion
        } else if (theTitle.Contains("pT")) {
```

I think `TString::Contains` is case-sensitive.

Reply:
There is a `ToLower()` ~80 lines above, at the beginning of the function `getTheRangeUser`.
https://github.com/cms-sw/cmssw/blob/a39a770a2cdf573772b1001b295c4db0a2cae520/Alignment/OfflineValidation/macros/FitPVResiduals.C#L4261
I think that it modifies `theTitle`, and the comparison should be correct.
I forgot to mention it in the PR description.

Reply:
ah yes, I added that 7 years ago: https://github.com/cms-sw/cmssw/commit/9c4c3330ee1ee0995272710c07300b08ee69efd3

Nonetheless can you add the newly added boundaries to the comment just above?

https://github.com/cms-sw/cmssw/blob/a39a770a2cdf573772b1001b295c4db0a2cae520/Alignment/OfflineValidation/macros/FitPVResiduals.C#L4263-L4282

Reply:
Done